README.md

```diff
@@ -1,48 +1,76 @@
+You can follow the tutorial [here](https://juju.is/docs/juju/set-up--tear-down-your-test-environment#heading--set-up-automatically) to set up a test environment for Juju with LXD.
+
+### Deploy
+From inside the virtual machine, deploy penpot charm's dependencies using the `juju deploy` command.
+
+```
+juju deploy minio --config access-key=minioadmin --config secret-key=minioadmin
+juju deploy postgresql-k8s --channel 14/stable --trust
+juju deploy redis-k8s --channel latest/edge
+juju deploy s3-integrator --config "endpoint=http://minio-endpoints.penpot-test.svc.cluster.local:9000" --config bucket=penpot
+juju deploy nginx-ingress-integrator --trust --config service-hostname=penpot.local --config path-routes=/
+juju deploy self-signed-certificates
+juju integrate self-signed-certificates nginx-ingress-integrator
+```
+
+Configure minio to provide a S3 compatible storage for the penpot charm.
 
-The Penpot Operator is a member of the Ubuntu family. It's an
-open source project that warmly welcomes community projects, contributions,
-suggestions, fixes and constructive feedback.
-* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
-* [Get support](https://discourse.charmhub.io/)
-<!--
-* [Contribute](https://charmhub.io/penpot/docs/contributing)
-* [Roadmap](https://charmhub.io/penpot/docs/roadmap)
--->
-Thinking about using Penpot for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
-
----
-<!--
-For further details, [see the charm's detailed documentation](https://charmhub.io/penpot/docs).
--->
+```
+export AWS_ACCESS_KEY_ID=minioadmin
+export AWS_SECRET_ACCESS_KEY=minioadmin
+export AWS_ENDPOINT_URL=http://$(juju status --format=json | jq -r '.applications.minio.units."minio/0".address'):9000
+aws s3 mb s3://penpot
+juju run s3-integrator/0 sync-s3-credentials --string-args access-key=minioadmin secret-key=minioadmin
+```
+
+Deploy the penpot charm and integrate the penpot charm with all its dependencies.
+
+```
+juju deploy penpot --channel latest/edge
+juju integrate penpot postgresql-k8s
+juju integrate penpot redis-k8s
+juju integrate penpot s3-integrator
+juju integrate penpot nginx-ingress-integrator
+```
+
+### Basic operations
+When the penpot charm has completed deployment and installation, you can access penpot from a browser.  
+First, we need to modify the `/etc/hosts` file to point the `penpot.local` domain to the IP address of the virtual machine.  
+After that, we can access the penpot instance in the browser using the address `https://penpot.local`.  
+Note that `https` is required for penpot to function. You may need to bypass the certificate security warning in the browser, as we are using a self-signed certificate.  
+
+Inside the virtual machine, run the following command to create a penpot account, and use the returned credentials to log in with this account.
+
+```
+juju run penpot/0 create-profile --string-args email=example@example.com fullname="John Doe"
+```
+
```

Comment:
Any there any integrations that are worth mentioning (in a separate section `## Integrations`, after the `## Get started` section)?

For instance, are the dependencies from the previous section required for penpot to work?

Reply:
As we discussed in our 1:1, I think you can mention the mandatory integrations in the `Integrations` section and then link to the Charmhub documentation to let the reader know about the full list of integrations

Reply:
Yes, updated with descriptions for all integrations. Thanks!